### ISSUE TITLE
Container crash after implementing processor!

### ISSUE THREAD

**Issue by MaximilianWiesmayr** [FriendsOfSylius/SyliusImportExportPlugin#198]
I am using macos with mojave 10.14.5

So I am currently using tha SyliusImportExportPlugin since the 25th of Juli. Until friday the 26th it worked out fine. But since I reinstalled it on Monday the 29th my php container crashes when I am reloading the page. The container log writes out this error message:

``` Fatal error: During class fetch: Uncaught ReflectionException: Class Symfony\Component\Serializer\Normalizer\NormalizerInterface not found in /srv/sylius/vendor/friendsofsymfony/rest-bundle/Serializer/Normalizer/FormErrorNormalizer.php:22
!!  Stack trace:
!!  #0 /srv/sylius/vendor/symfony/debug/DebugClassLoader.php(160): require('/srv/sylius/ven...')
!!  #1 [internal function]: Symfony\Component\Debug\DebugClassLoader->loadClass('FOS\\RestBundle\\...')
!!  #2 [internal function]: spl_autoload_call('FOS\\RestBundle\\...')
!!  #3 /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php(78): class_exists('FOS\\RestBundle\\...')
!!  #4 /srv/sylius/vendor/symfony/dependency-injection/ContainerBuilder.php(353): Symfony\Component\Config\Resource\ClassExistenceResource->isFresh(0)
!!  #5 /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php(336): Symfony\Component\DependencyInjection\ContainerBuilder->getReflectionClass('FOS\\RestBundle\\...', false)
!!  #6 /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.ph in /srv/sylius/vendor/friendsofsymfony/rest-bundle/Serializer/Normalizer/FormErrorNormalizer.php on line 22
!!  
!!  Call Stack:
!!      0.0013     397224   1. {main}() /srv/sylius/bin/console:0
!!      0.2207    2341120   2. Symfony\Bundle\FrameworkBundle\Console\Application->run() /srv/sylius/bin/console:38
!!      0.2445    2429568   3. Symfony\Bundle\FrameworkBundle\Console\Application->doRun() /srv/sylius/vendor/symfony/console/Application.php:149
!!      0.2445    2429568   4. Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands() /srv/sylius/vendor/symfony/framework-bundle/Console/Application.php:65
!!      0.2445    2429568   5. App\Kernel->boot() /srv/sylius/vendor/symfony/framework-bundle/Console/Application.php:159
!!      0.3291    2733408   6. App\Kernel->initializeContainer() /srv/sylius/vendor/symfony/http-kernel/Kernel.php:133
!!      4.4206    5544904   7. Symfony\Component\DependencyInjection\ContainerBuilder->compile() /srv/sylius/vendor/symfony/http-kernel/Kernel.php:550
!!      4.6660    5548848   8. Symfony\Component\DependencyInjection\Compiler\Compiler->compile() /srv/sylius/vendor/symfony/dependency-injection/ContainerBuilder.php:756
!!     22.7953   24322992   9. Symfony\Component\DependencyInjection\Compiler\AutowirePass->process() /srv/sylius/vendor/symfony/dependency-injection/Compiler/Compiler.php:95
!!     22.7953   24322992  10. Symfony\Component\DependencyInjection\Compiler\AutowirePass->process() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:52
!!     22.7953   24322992  11. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:46
!!     22.7954   24322992  12. Symfony\Component\DependencyInjection\Compiler\AutowirePass->doProcessValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:71
!!     22.7954   24322992  13. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:99
!!     22.7976   24322992  14. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:82
!!     22.7976   24322992  15. Symfony\Component\DependencyInjection\Compiler\AutowirePass->doProcessValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:71
!!     22.7981   24323896  16. Symfony\Component\DependencyInjection\Compiler\AutowirePass->autowireCalls() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:122
!!     22.7981   24323896  17. Symfony\Component\DependencyInjection\Compiler\AutowirePass->autowireMethod() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:172
!!     22.7984   24326152  18. Symfony\Component\DependencyInjection\Compiler\AutowirePass->Symfony\Component\DependencyInjection\Compiler\{closure:/srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:230-242}() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:262
!!     22.8761   24548912  19. Symfony\Component\DependencyInjection\Exception\AutowiringFailedException->__construct() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:237
!!     22.8761   24548912  20. Symfony\Component\DependencyInjection\Compiler\AutowirePass->Symfony\Component\DependencyInjection\Compiler\{closure:/srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:383-385}() /srv/sylius/vendor/symfony/dependency-injection/Exception/AutowiringFailedException.php:27
!!     22.8761   24548912  21. Symfony\Component\DependencyInjection\Compiler\AutowirePass->createTypeNotFoundMessage() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:384
!!     22.8763   24549048  22. Symfony\Component\DependencyInjection\Compiler\AutowirePass->createTypeAlternatives() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:403
!!     22.8764   24549048  23. Symfony\Component\DependencyInjection\Compiler\AutowirePass->populateAvailableTypes() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:429
!!     25.0072   31056240  24. Symfony\Component\DependencyInjection\Compiler\AutowirePass->populateAvailableType() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:322
!!     25.0072   31056240  25. Symfony\Component\DependencyInjection\ContainerBuilder->getReflectionClass() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:336
!!     25.0073   31056320  26. Symfony\Component\Config\Resource\ClassExistenceResource->isFresh() /srv/sylius/vendor/symfony/dependency-injection/ContainerBuilder.php:353
!!     25.0073   31056504  27. class_exists() /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:78
!!     25.0074   31056600  28. spl_autoload_call() /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:78
!!     25.0074   31056696  29. Symfony\Component\Debug\DebugClassLoader->loadClass() /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:78
!!     25.0083   31057536  30. require('/srv/sylius/vendor/friendsofsymfony/rest-bundle/Serializer/Normalizer/FormErrorNormalizer.php') /srv/sylius/vendor/symfony/debug/DebugClassLoader.php:160
!!  
!!  
!!  In FormErrorNormalizer.php line 22:
!!                                                                                 
!!    Error: During class fetch: Uncaught ReflectionException: Class Symfony\Comp  
!!    onent\Serializer\Normalizer\NormalizerInterface not found in /srv/sylius/ve  
!!    ndor/friendsofsymfony/rest-bundle/Serializer/Normalizer/FormErrorNormalizer  
!!    .php:22                                                                      
!!    Stack trace:                                                                 
!!    #0 /srv/sylius/vendor/symfony/debug/DebugClassLoader.php(160): require('/sr  
!!    v/sylius/ven...')                                                            
!!    #1 [internal function]: Symfony\Component\Debug\DebugClassLoader->loadClass  
!!    ('FOS\\RestBundle\\...')                                                     
!!    #2 [internal function]: spl_autoload_call('FOS\\RestBundle\\...')            
!!    #3 /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php(78  
!!    ): class_exists('FOS\\RestBundle\\...')                                      
!!    #4 /srv/sylius/vendor/symfony/dependency-injection/ContainerBuilder.php(353  
!!    ): Symfony\Component\Config\Resource\ClassExistenceResource->isFresh(0)      
!!    #5 /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.ph  
!!    p(336): Symfony\Component\DependencyInjection\ContainerBuilder->getReflecti  
!!    onClass('FOS\\RestBundle\\...', false)                                       
!!    #6 /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.ph
```

Note: I am getting the same error message when I am rebuilding my application.

Following the relevant Code that I added to cause this error:

bundles.php:

```
<?php

use FriendsOfSylius\SyliusImportExportPlugin\FOSSyliusImportExportPlugin;

return [
.
.
.
    FriendsOfSylius\SyliusImportExportPlugin\FOSSyliusImportExportPlugin::class => ['all' => true],
];
```

fos_sylius_import_export.yaml:
```
fos_sylius_import_export:
    importer:
        # set to false to not add an upload form to the entity overview pages
        web_ui:               true
        # set to an integer value bigger than 0 to flush the object manager in regular intervals
        batch_size:           0
        # if incomplete rows (ie. missing required fields) should be considered failures
        fail_on_incomplete:   true
        # if to stop the import process in case of a failure
        stop_on_failure:      false
    exporter:
        # set to false to not add export buttons
        web_ui:               true
    message_queue:
        service_id: 'redis_connection_factory'
```

service_importexportplugin.yaml:
```
services:

    redis_connection_factory:
        class: Enqueue\Redis\RedisConnectionFactory
              
    sylius.importer.customer.csv:
        class: FriendsOfSylius\SyliusImportExportPlugin\Importer\ResourceImporter
        arguments:
            - "@sylius.factory.csv_reader"
            - "@sylius.manager.customer"
            - "@sylius.processor.customer"
            - "@sylius.importer.result"
            - "%sylius.importer.batch_size%"
            - "%sylius.importer.fail_on_incomplete%"
            - "%sylius.importer.stop_on_failure%"
        tags:
            - { name: sylius.importer, type: customer, format: csv }          

    sylius.processor.customer:
        class: App\Processor\CustomerProcessor
        arguments:
            - '@sylius.factory.customer_after_checkout'
            - '@sylius.repository.customer'
            - '@sylius.importer.metadata_validator'
            - '@doctrine.orm.entity_manager'
            - ["id", "customer_group_id", "email", "first_name", "last_name", "birthday", "gender", "phone_number", "since", "title", "verificationCode", "hasApp", "street", "street_number", "postcode", "city", "opt_in_post", "opt_in_phone", "opt_in_email"]

```

CustomerProcessor.php:
```
<?php
namespace App\Processor;


use App\Entity\Customer\Customer;
use Doctrine\ORM\EntityManagerInterface;
use FriendsOfSylius\SyliusImportExportPlugin\Exception\ImporterException;
use Payum\Core\Model\GatewayConfigInterface;
use Sylius\Component\Core\Factory\CustomerAfterCheckoutFactoryInterface;
use Sylius\Component\Core\Model\CustomerInterface;
use Sylius\Component\Resource\Repository\RepositoryInterface;
use FriendsOfSylius\SyliusImportExportPlugin\Processor\ResourceProcessorInterface;
use FriendsOfSylius\SyliusImportExportPlugin\Processor\MetadataValidatorInterface;
class CustomerProcessor implements ResourceProcessorInterface
{
    /** @var CustomerAfterCheckoutFactoryInterce */
    private $resourceFactory;
    /** @var RepositoryInterface */
    private $resourceRepository;
    /** @var MetadataValidatorInterface */
    private $metadataValidator;
    /** @var EntityManagerInterface */
    private $entityManager;
    /** @var string[] */
    private $headerKeys;
    /**
     * @param string[] $headerKeys
     */
    public function __construct(
        CustomerAfterCheckoutFactoryInterface $factory,
        RepositoryInterface $repository,
        MetadataValidatorInterface $metadataValidator,
        EntityManagerInterface $entityManager,
        array $headerKeys
    ) {
        $this->resourceFactory = $factory;
        $this->resourceRepository = $repository;
        $this->metadataValidator = $metadataValidator;
        $this->entityManager = $entityManager;
        $this->headerKeys = $headerKeys;
    }
    public function process(array $data): void
    {
        $entityManager = $this->getDoctrine()->getManager();
        var_dump($entityManager);
        $customerRepository = $this->entityManager->getRepository(Customer::class);
        $customerRepository->findOneBy(['customerNumber']);
        var_dump($customerRepository);
        // existiert der Customer mit der customerNumber 410 ($data['customer_id']) bereits?
        // customerNumber existiert bereits: Customer Daten mit Daten aus data array überschreiben
        // Customer Daten persistieren und flushen
    }

}

```

routes.yaml:
```
sylius_import_export:
    resource: "@FOSSyliusImportExportPlugin/Resources/config/routing.yml"
    prefix: /admin

```

Error message after I removed this line: ` use FriendsOfSylius\SyliusImportExportPlugin\FOSSyliusImportExportPlugin; ` from Bundles.php:
```

!!  In FileLoader.php line 166:
!!                                                                                 
!!    There is no extension able to load the configuration for "fos_sylius_import  
!!    _export" (in /srv/sylius/config/packages/fos_sylius_import_export.yaml). Lo  
!!    oked for namespace "fos_sylius_import_export", found "framework", "monolog"  
!!    , "security", "swiftmailer", "twig", "doctrine", "doctrine_cache", "sylius_  
!!    order", "sylius_money", "sylius_currency", "sylius_locale", "sylius_product  
!!    ", "sylius_channel", "sylius_attribute", "sylius_taxation", "sylius_shippin  
!!    g", "sylius_payment", "sylius_mailer", "sylius_promotion", "sylius_addressi  
!!    ng", "sylius_inventory", "sylius_taxonomy", "sylius_user", "sylius_customer  
!!    ", "sylius_ui", "sylius_review", "sylius_core", "sylius_resource", "sylius_  
!!    grid", "winzou_state_machine", "sonata_core", "sonata_block", "sonata_intl"  
!!    , "bazinga_hateoas", "jms_serializer", "fos_rest", "knp_gaufrette", "knp_me  
!!    nu", "liip_imagine", "payum", "stof_doctrine_extensions", "white_october_pa  
!!    gerfanta", "doctrine_migrations", "doctrine_fixtures", "sylius_fixtures", "  
!!    sylius_payum", "sylius_theme", "sylius_admin", "sylius_shop", "fos_oauth_se  
!!    rver", "sylius_admin_api" in /srv/sylius/config/packages/fos_sylius_import_  
!!    export.yaml (which is loaded in resource "/srv/sylius/config/packages/fos_s  
!!    ylius_import_export.yaml").                                                  
!!                                                                                 
!!  
!!  In YamlFileLoader.php line 683:
!!                                                                                 
!!    There is no extension able to load the configuration for "fos_sylius_import  
!!    _export" (in /srv/sylius/config/packages/fos_sylius_import_export.yaml). Lo  
!!    oked for namespace "fos_sylius_import_export", found "framework", "monolog"  
!!    , "security", "swiftmailer", "twig", "doctrine", "doctrine_cache", "sylius_  
!!    order", "sylius_money", "sylius_currency", "sylius_locale", "sylius_product  
!!    ", "sylius_channel", "sylius_attribute", "sylius_taxation", "sylius_shippin  
!!    g", "sylius_payment", "sylius_mailer", "sylius_promotion", "sylius_addressi  
!!    ng", "sylius_inventory", "sylius_taxonomy", "sylius_user", "sylius_customer  
!!    ", "sylius_ui", "sylius_review", "sylius_core", "sylius_resource", "sylius_  
!!    grid", "winzou_state_machine", "sonata_core", "sonata_block", "sonata_intl"  
!!    , "bazinga_hateoas", "jms_serializer", "fos_rest", "knp_gaufrette", "knp_me  
!!    nu", "liip_imagine", "payum", "stof_doctrine_extensions", "white_october_pa  
!!    gerfanta", "doctrine_migrations", "doctrine_fixtures", "sylius_fixtures", "  
!!    sylius_payum", "sylius_theme", "sylius_admin", "sylius_shop", "fos_oauth_se  
!!    rver", "sylius_admin_api"

```

Error message after downgrading the SyliusandFriends/ImportExportPlugin version to 0.16.0:

```
!!  Fatal error: During class fetch: Uncaught ReflectionException: Class Symfony\Component\Serializer\Normalizer\NormalizerInterface not found in /srv/sylius/vendor/friendsofsymfony/rest-bundle/Serializer/Normalizer/FormErrorNormalizer.php:22
!!  Stack trace:
!!  #0 /srv/sylius/vendor/composer/ClassLoader.php(444): include('/srv/sylius/ven...')
!!  #1 /srv/sylius/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile('/srv/sylius/ven...')
!!  #2 [internal function]: Composer\Autoload\ClassLoader->loadClass('FOS\\RestBundle\\...')
!!  #3 [internal function]: spl_autoload_call('FOS\\RestBundle\\...')
!!  #4 /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php(78): class_exists('FOS\\RestBundle\\...')
!!  #5 /srv/sylius/vendor/symfony/dependency-injection/ContainerBuilder.php(353): Symfony\Component\Config\Resource\ClassExistenceResource->isFresh(0)
!!  #6 /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php(336): Symfony\Component\DependencyInjection\ContainerBuilder->getReflectionClass('FOS\\RestBundle\\...' in /srv/sylius/vendor/friendsofsymfony/rest-bundle/Serializer/Normalizer/FormErrorNormalizer.php on line 22
!!  
!!  Call Stack:
!!      0.0002     396216   1. {main}() /srv/sylius/bin/console:0
!!      0.0888    5471344   2. Symfony\Bundle\FrameworkBundle\Console\Application->run() /srv/sylius/bin/console:38
!!      0.0966    5552496   3. Symfony\Bundle\FrameworkBundle\Console\Application->doRun() /srv/sylius/vendor/symfony/console/Application.php:149
!!      0.0966    5552496   4. Symfony\Bundle\FrameworkBundle\Console\Application->registerCommands() /srv/sylius/vendor/symfony/framework-bundle/Console/Application.php:65
!!      0.0966    5552496   5. App\Kernel->boot() /srv/sylius/vendor/symfony/framework-bundle/Console/Application.php:159
!!      0.1104    5806400   6. App\Kernel->initializeContainer() /srv/sylius/vendor/symfony/http-kernel/Kernel.php:133
!!      2.6877    8477512   7. Symfony\Component\DependencyInjection\ContainerBuilder->compile() /srv/sylius/vendor/symfony/http-kernel/Kernel.php:550
!!      2.7258    8481456   8. Symfony\Component\DependencyInjection\Compiler\Compiler->compile() /srv/sylius/vendor/symfony/dependency-injection/ContainerBuilder.php:756
!!     17.1914   25422480   9. Symfony\Component\DependencyInjection\Compiler\AutowirePass->process() /srv/sylius/vendor/symfony/dependency-injection/Compiler/Compiler.php:95
!!     17.1914   25422480  10. Symfony\Component\DependencyInjection\Compiler\AutowirePass->process() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:52
!!     17.1915   25422480  11. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:46
!!     17.1915   25422480  12. Symfony\Component\DependencyInjection\Compiler\AutowirePass->doProcessValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:71
!!     17.1915   25422480  13. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:99
!!     17.1933   25422480  14. Symfony\Component\DependencyInjection\Compiler\AutowirePass->processValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AbstractRecursivePass.php:82
!!     17.1933   25422480  15. Symfony\Component\DependencyInjection\Compiler\AutowirePass->doProcessValue() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:71
!!     17.1937   25423384  16. Symfony\Component\DependencyInjection\Compiler\AutowirePass->autowireCalls() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:122
!!     17.1938   25423384  17. Symfony\Component\DependencyInjection\Compiler\AutowirePass->autowireMethod() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:172
!!     17.1940   25425640  18. Symfony\Component\DependencyInjection\Compiler\AutowirePass->Symfony\Component\DependencyInjection\Compiler\{closure:/srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:230-242}() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:262
!!     17.2524   25566480  19. Symfony\Component\DependencyInjection\Exception\AutowiringFailedException->__construct() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:237
!!     17.2524   25566480  20. Symfony\Component\DependencyInjection\Compiler\AutowirePass->Symfony\Component\DependencyInjection\Compiler\{closure:/srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:383-385}() /srv/sylius/vendor/symfony/dependency-injection/Exception/AutowiringFailedException.php:27
!!     17.2524   25566480  21. Symfony\Component\DependencyInjection\Compiler\AutowirePass->createTypeNotFoundMessage() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:384
!!     17.2525   25566616  22. Symfony\Component\DependencyInjection\Compiler\AutowirePass->createTypeAlternatives() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:403
!!     17.2526   25566616  23. Symfony\Component\DependencyInjection\Compiler\AutowirePass->populateAvailableTypes() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:429
!!     17.7874   31313064  24. Symfony\Component\DependencyInjection\Compiler\AutowirePass->populateAvailableType() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:322
!!     17.7874   31313064  25. Symfony\Component\DependencyInjection\ContainerBuilder->getReflectionClass() /srv/sylius/vendor/symfony/dependency-injection/Compiler/AutowirePass.php:336
!!     17.7875   31313144  26. Symfony\Component\Config\Resource\ClassExistenceResource->isFresh() /srv/sylius/vendor/symfony/dependency-injection/ContainerBuilder.php:353
!!     17.7875   31313328  27. class_exists() /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:78
!!     17.7875   31313424  28. spl_autoload_call() /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:78
!!     17.7876   31313520  29. Composer\Autoload\ClassLoader->loadClass() /srv/sylius/vendor/symfony/config/Resource/ClassExistenceResource.php:78
!!     17.7876   31313520  30. Composer\Autoload\includeFile() /srv/sylius/vendor/composer/ClassLoader.php:322
!!     17.7878   31314200  31. include('/srv/sylius/vendor/friendsofsymfony/rest-bundle/Serializer/Normalizer/FormErrorNormalizer.php') /srv/sylius/vendor/composer/ClassLoader.php:444


```

Versions and more:

I am using Sylius via Docker (vers.: 2.1.0.0, stable)
php-version: 7.3
Sylius version: 1.5
SyliusandFriends/ImportExportPlugin version: 0.17.0

**Comment by oallain** [comment]
Hi @MaximilianWiesmayr ,

Can you test with :

- remove line  `use FriendsOfSylius...` into bundles.php
- clear cache
- => copy/paste log in this issue if it fail

If you have yet error, can you test :

- force version of SyliusandFriends/ImportExportPlugin to 0.16.0 in your composer.json
- composer update
- cache clear
- => copy/paste log in this issue if it fail

Thanks

**Comment by MaximilianWiesmayr** [comment]
Ok, I updated everything for you :)

**Comment by MaximilianWiesmayr** [comment]
Ok, I added it to this issue.

**Comment by MaximilianWiesmayr** [comment]
Oh I am sorry, I forgot to mention that I am using sylius with docker. I added it to the issue.

**Comment by MaximilianWiesmayr** [comment]
I just installed and added everything completely new and somehow it worked out. Apparently I didn't do anything entirely different and I can't imagine a reason for this whole error. So I don't know if this happens again or if this was an one time error. I will leave it to you if this issue should stay open or get closed. 

The only maybe relevant difference was that I used linux for setting it up.
